samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties

```diff
@@ -36,3 +36,21 @@ action_save_pair=Gorde parea
 delete=Ezabatu
 delete_attempt=Ezabatu saialdia
 confirm_delete_attempt=Ziur zaude saialdi hau ezabatu nahi duzula?
+
+# S2U-14
+cancel_question=Cancel question
```

Comment:
Hi @PabloSanRoman, could you help me with the EU translations please?

Reply:
Hello @mpellicer !!
+ # S2U-14
+ cancel_question=Ezabatu galdera

Reply:
Thanks Pablo! Are you going to contribute translations for the rest of the strings?

Reply:
Hi @mpellicer , I'm sorry, the strings in the comment are not displayed, they can be seen opening the link of _eu.properties. Anyway, here you are: 

cancel_question=Ezabatu galdera
cancel_question_reduce_total=Murriztu puntuazio osoa
cancel_question_distribute_points=Modu berean banatu puntuak
cancel_question_cancel=Ez ezabatu galdera
cancel_question_info_cancelled_question=Galdera hau bertan behera geratu da eta ez du eraginik izango puntuazio osoan.
cancel_question_info_cancellation=Galdera hori bertan behera uzten baduzu kalifikazioan ez da kontuan hartuko. \
Ezabatzearen eragina bi modutan kudeatu daiteke:
cancel_question_info_distribute_points=Ezabatutako galderaren puntuak berdin banatuko dira gainerako galderen artean.\
Puntuazio osoan eraginik ez dago.
cancel_question_info_reduce_total=Ezabatutako galderaren puntuak puntuazio osotik kenduko dira.\
Gainerako galderetan eraginik ez dago. 
cancel_question_info_regrade=Galdera ezabatutakoan egin diren bidalketak berriro kalifikatuko dira.
cancel_question_info_no_undo=Ekintza hau ezin da berriz desegin!
cancel_question_info_skip_question=Galdera hau saihestu dezakezu.
cancel_question_error_cancelling=Errore bat gertatu da galdera ezabatzerakoan. Ezer ez da aldatu.
cancel_question_info_emi=Kalifikazio doiturik ez dute jasoko "Lotzeko aurreratua" motako elementuek.

Reply:
No problem, thank you!